### PR TITLE
Feature/redesign bottombar

### PR DESCRIPTION
**Pull request by mayo-s**
Removed old fab circular menu and added bottom bar as component

Edit by Philipp:
* renamed some files
* deleted unused routes and files
* fixed issue with static map parameter

### CODE REVIEW

File: lib/pages/home.dart [hunk 1]
```diff
@@ -23,62 +20,16 @@ class _HomeState extends State<Home> {
...
-              iconSize: 42.0
-            ),
-            IconButton(icon: Icon(
-              Icons.settings),
-              onPressed: () { Navigator.pushNamed(context, '/settings');},
-              iconSize: 42.0
-            ),
-          ],
-        ),
-          Align(
```
Comment (Capscreen) [inline lib/pages/home.dart:57]: Please note, that this was a test case you just deleted. It is not necessary to keep this one, but in the future nobody will be able to test this feature.

File: lib/pages/settings/settings.dart [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import 'package:flutter/material.dart';
+import 'package:hiking4nerds/services/help_data.dart';
+
+class Settings extends StatefulWidget {
+  @override
+  _SettingsState createState() => _SettingsState();
+}
+
+class _SettingsState extends State<Settings> {
+
```
Comment (Capscreen) [inline lib/pages/settings/settings.dart:10]: A localization TODO would be nice here

**Review by Capscreen — COMMENTED** [review]
please note my comments.

File: lib/pages/home.dart (hunk 1 above)
Comment (dephiloper) [reply lib/pages/home.dart:57]: you are totally right, my bad

**Review by dephiloper — APPROVED** [review]

**Pull request merged**